lib/commands/doctor/checks/logged-in-user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const execa = require('execa');
+const errors = require('../../../errors');
+const chalk = require('chalk');
+const os = require('os');
+
+const taskTitle = 'Checking logged in user';
+
+function loggedInUser() {
+    const uid = process.getuid();
+    let ghostuid;
+
+    try {
+        ghostuid = execa.shellSync('id -u ghost').stdout;
+    } catch (e) {
+        // CASE: the ghost user doesn't exist yet, all good in the hood
+        return Promise.resolve();
```

Comment:
this can just be `return` -> there's no async behavior anywhere in the function so no need to make it async.